examples/model_service_template/mxnet_vision_batching.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import mxnet as mx
+import json
+import os
+import numpy as np
+from collections import namedtuple
+import logging
+
+
+class MXNetVisionServiceBatching(object):
```

Comment:
Can you just rename this file to mxnet_vision_service.py?
So we only maintain one version of vision service.

Reply:
I wanted to do that, but I thought we would need discussions before I merged this into mxnet_vision_service.py . Shall I do that with a separate PR. I also plan to write a one pager to go over the pro's and con's of merging these two together.